neuromation/api/jobs.py

```diff
@@ -361,8 +361,13 @@ def _http_port_from_api(data: Dict[str, Any]) -> HTTPPort:
 
 
 def _container_from_api(data: Dict[str, Any], parser: _ImageNameParser) -> Container:
+    try:
+        image = parser.parse_remote(data["image"])
+    except ValueError:
+        image = RemoteImage(name=f'Invalid Image Name {data["image"]}')
```

Comment:
I'm curious how this super-long-line looks in tables view provided by `TabularJobsFormatter`?

Reply:
It is why I propose to add new Class, then we can use different formatting for `ps` and `status`.
I.e. `InvalidImage` for `ps` and `Invalid Image Name [Name here]` for `ps`

Reply:
Do you really want to print this invalid image name?
Now you cannot create such a thing, the only way to get invalid image is retrieving historical old enough job. Missing exact invalid image name doesn't make much harm.
I don't insist though. If you prefer your version -- please go ahead and merge.

The very minor and rare thing anyway.

Reply:
Agree, will change